app/views/kiosk/index.html.erb

```diff
@@ -3,4 +3,29 @@
      data-kiosk-type="<%= @kiosk_type %>"
      data-slides='<%= raw(JSON.dump @slides) %>'
      data-kiosk-url="<%= kiosk_url id: @kiosk_type %>"
```

Comment:
Can you remove this DIV up top, and just have your H1 and the table below?
Also add the login form if the user isn't already logged in?